pkg/ottl/ottlfuncs/README.md

```diff
@@ -220,6 +221,35 @@ Examples:
 
 - `limit(resource.attributes, 50, ["http.host", "http.method"])`
 
+## parse_json_into_map
+
+`parse_json_into_map(target, value)`
```

Comment:
Just an idea: I think we will end with multiple parseIntoMap that accepts multiple formats, JSON/YAML/etc. Do you think it is worth accepting the format as argument instead?